src/main/resources/common-services/CDAP/4.0.0/package/scripts/ambari_helpers.py

```diff
@@ -41,8 +41,12 @@ def add_repo(source, dest):
     Execute("rm -f %s" % (dest_file))
     # Skip sed if CDAP repos exist, we're on a newer Ambari... yay!
     no_op_test = format('ls {dest} 2>/dev/null | grep CDAP >/dev/null 2>&1')
+    # default gpg check to true
+    gpgcheck = 1
+    if params.gpgcheck_enabled:
+        gpg = 0
```

Comment:
should this be `gpgcheck` ?  if so, should it be `gpgcheck = 1` and the above line default to `0`?  (otherwise i dont see how the converse case works)